docs/articles/preamble.md

```diff
@@ -3,7 +3,7 @@ uid: articles.preamble
 title: Article Preamble
 ---
 
-> [!IMPORTANT]
```

Comment:
This is intentionally a big red box and even now many people dont read it

Reply:
Perhaps the wording of that could be improved too.

Reply:
i do think red triggers more of a "this is important" connotation, but, no strong opinions